internal/controllers/common_controller.go

```diff
@@ -55,51 +88,106 @@ func setInstalledStatusConditionUnknown(conditions *[]metav1.Condition, message
 	})
 }
 
-// setResolvedStatusConditionFailed sets the resolved status condition to failed.
-func setResolvedStatusConditionFailed(conditions *[]metav1.Condition, message string, generation int64) {
+// setProgressingStatusConditionFalse sets the progressing status condition to false, awaiting next reconcile.
+func setProgressingStatusConditionFalse(conditions *[]metav1.Condition, message string, generation int64) {
 	apimeta.SetStatusCondition(conditions, metav1.Condition{
-		Type:               ocv1alpha1.TypeResolved,
+		Type:               ocv1alpha1.TypeProgressing,
 		Status:             metav1.ConditionFalse,
-		Reason:             ocv1alpha1.ReasonResolutionFailed,
+		Reason:             ocv1alpha1.ReasonPending,
```

Comment:
If we're not progressing, doesn't that mean we're at steady state with nothing to do? `Pending` seems like it would indicate that we _are_ progressing and waiting on some action to occur.

Reply:
+1. If it is assumed that when `Progressing` is `False` we are in a stable state maybe `Stable` as the reason is sufficient?